autogen/logger/__init__.py

```diff
@@ -1,4 +1,5 @@
+from .cosmos_db_logger import CosmosDBLogger
 from .logger_factory import LoggerFactory
 from .sqlite_logger import SqliteLogger
 
-__all__ = ("LoggerFactory", "SqliteLogger")
+__all__ = ("LoggerFactory", "SqliteLogger", "CosmosDBLogger")
```

Comment:
can you also mention the FileLogger. Thanks

Reply:
@Hk669  Feel free to suggest edits

Reply:
@Hk669 Feel free to suggest the exact edits

Reply:
```suggestion
__all__ = ("LoggerFactory", "SqliteLogger", "FileLogger", "CosmosDBLogger")
```

Reply:
dont worry, ill make a new PR. Thanks @wmwxwa